libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -57,12 +57,12 @@ export class OrchestratorAdaptiveRecognizer extends Recognizer implements Orches
     /**
      * Path to Orchestrator base model folder.
      */
-    public modelPath: StringExpression = new StringExpression('=settings.orchestrator.modelPath');
+    public modelFolder: StringExpression = new StringExpression('=settings.orchestrator.modelFolder');
```

Comment:
Not sure if it matters - but the casing is different from the schema file - `modelfolder` in schema and `modelFolder` here. Same as the snapshotFile settings on line 65.